docs/advanced-features/using-mdx.md

```diff
@@ -64,6 +85,14 @@ The following steps outline how to setup `@next/mdx` in your Next.js project:
      - package.json
    ```
 
+## next-mdx-remote
+
+On a remote database, use next-mdx-remote https://github.com/hashicorp/next-mdx-remote
+
+## @mdx-js/mdx
+
+Handle mdx strings client side, probably @mdx-js/mdx see official docs https://mdxjs.com/docs/getting-started/
```

Comment:
Can we revert this or fix forward? This is not a complete sentence or using a correct link.

Reply:
Yeah I think we should revert based on my other comment.

Reply:
Yeah it needs to be fixed